src/main/java/org/broadinstitute/dsm/db/dao/fieldsettings/FieldSettingsDao.java

```diff
@@ -177,21 +177,20 @@ public int delete(int id) {
                 stmt.setString(2, columnName);
                 try(ResultSet fieldSettingsByColumnNameRs = stmt.executeQuery()) {
                     if (fieldSettingsByColumnNameRs.next()) {
-                        dbVals.resultValue = new FieldSettingsDto(
-                                fieldSettingsByColumnNameRs.getInt(FIELD_SETTINGS_ID),
-                                fieldSettingsByColumnNameRs.getInt(DDP_INSTANCE_ID),
-                                fieldSettingsByColumnNameRs.getString(FIELD_TYPE),
-                                fieldSettingsByColumnNameRs.getString(COLUMN_NAME),
-                                fieldSettingsByColumnNameRs.getString(COLUMN_DISPLAY),
-                                fieldSettingsByColumnNameRs.getString(DISPLAY_TYPE),
-                                fieldSettingsByColumnNameRs.getString(POSSIBLE_VALUES),
-                                fieldSettingsByColumnNameRs.getString(ACTIONS),
-                                fieldSettingsByColumnNameRs.getBoolean(READONLY),
-                                fieldSettingsByColumnNameRs.getInt(ORDER_NUMBER),
-                                fieldSettingsByColumnNameRs.getBoolean(DELETED),
-                                fieldSettingsByColumnNameRs.getLong(LAST_CHANGED),
-                                fieldSettingsByColumnNameRs.getString(CHANGED_BY)
-                        );
+                        dbVals.resultValue = new FieldSettingsDto.Builder(fieldSettingsByColumnNameRs.getInt(DDP_INSTANCE_ID))
+                                .withFieldSettingsId(fieldSettingsByColumnNameRs.getInt(FIELD_SETTINGS_ID))
```

Comment:
I think the order of field_settings_id and ddp_instance_id  in FieldSettingsDto is not correct here

Reply:
Builder pattern is used here, in constructor of Builder takes `ddp_instance_id` and ordering for helper methods, which start `with` prefix, does not matter.

Reply:
Oh I missed the Builder constructor having instance_id :doh: